{% extends "zerver/portico.html" %}

{% block title %}
<title>{{ _("Authentication subdomain error") }} | OneHash Connect</title>
{% endblock %}

{% block portico_content %}

<div class="error_page">
    <div class="container">
        <div class="row-fluid">
            <img src="{{ static('images/errors/500art.svg') }}" alt=""/>
            <div class="errorbox">
                <div class="errorcontent">
                    <h1 class="lead">{{ _("Authentication subdomain") }}</h1>
                    <p>
                        {% trans %}
                        It appears you ended up here by accident. This site
                        is meant to be an intermediate step in the authentication process
                        and shouldn't be accessed manually. If you came here directly,
                        you probably got the address wrong. If you got stuck here while trying
                        to log in, this is most likely a server bug or misconfiguration.
                        {% endtrans %}
                    </p>
                </div>
            </div>
        </div>
    </div>
</div>

{% endblock %}
